spec/requests/create_collection_spec.rb

```diff
@@ -116,7 +116,7 @@
             'release_date(1i)' => '2020',
             'release_date(2i)' => '7',
             'release_date(3i)' => '14',
-            'release_duration' => '1 month',
```

Comment:
This test uses a value which no longer allowed, I just changed it to an allowed value.  The actual value used is unimportant for the test itself, just needs to be balid.

Reply:
OK, no problem. And I'm not sure this PR really needs other testing, since it is a unification of existing code paths w/ some elimination of values.